test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/FileExtensionsAttributeAdapterTest.cs

```diff
@@ -15,26 +15,60 @@ namespace Microsoft.AspNetCore.Mvc.ModelBinding.Validation
     public class FileExtensionsAttributeAdapterTest
     {
         [Theory]
-        [InlineData("jpg,png")]
-        [InlineData("jpg, png")]
-        [InlineData("JPEG, Png")]
-        [ReplaceCulture]
```

Comment:
Doesn't this method also need `[ReplaceCulture]` since data annotations is localized?